javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/ignore/UserExcludedClassloadersConfigurer.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.tooling.ignore;
+
+import static java.util.Collections.emptyList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesBuilder;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesConfigurer;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import java.util.List;
+
+@AutoService(IgnoredTypesConfigurer.class)
+public class UserExcludedClassloadersConfigurer implements IgnoredTypesConfigurer {
```

Comment:
or preferred spelling is `class loader`, change class name to `UserExcludedClassLoadersConfigurer`. Also change the variables and config property accordingly.

Reply:
You totally misunderstood the ask was to use `ClassLoaders` instead of `Classloaders` (note the capitalized `L` in the middle)